packages/lib/server/repository/eventType.ts

```diff
@@ -156,6 +156,13 @@ export class EventTypeRepository {
             {
               profileId,
             },
+            // Fetch children event-types by userId because profileId is wrong
```

Comment:
profileId is wrong for children event-types. Will open a separate PR to fix that and add a migration to fix existing data.